partiql-tests-runner/build.gradle

```
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *       http://aws.amazon.com/apache2.0/
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ *  language governing permissions and limitations under the License.
+ */
+
+plugins {
+    id 'java-library'
+}
+
+task wrapper(type: Wrapper){
+   gradleVersion = '7.5.1'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+
+    testImplementation("org.jetbrains.kotlin:kotlin-test")
+    testImplementation("org.junit.jupiter:junit-jupiter-params:5.8.2")
+
+    testImplementation("org.partiql:partiql-lang-kotlin:0.7.0")
```

For now, we fix the `partiql-lang-kotlin` version to the latest version in Maven. In the future, we will likely want this to point to the version in `lang`

---

Why for now? I think this is the right opportunity to fix the gradle setup. 
```suggestion
    testImplementation(project(":lang"))
```